pytest_container/container.py

```diff
@@ -619,7 +619,7 @@ def filelock_filename(self) -> str:
                 all_elements.append("".join(value.values()))
             else:
                 all_elements.append(str(value))
-        return f"{md5((''.join(all_elements)).encode()).hexdigest()}.lock"
+        return f"{sha256((''.join(all_elements)).encode()).hexdigest()}.lock"
```

Comment:
you can also simply pass `usedforsecurity=False` which reenables md5

Reply:
Bummer, this option is not available for old pythons :-(